src/assets/css/contributors.scss

```diff
@@ -68,120 +69,154 @@
   max-width: 1200px;
   margin: 0 auto;
   padding: 0 1rem;
-  @include media.md {
+  @include media.sm {
+    padding-top: 1rem;
+    overflow: hidden;
+  }
+  @include media.md-rem {
     padding: 0;
+    overflow: inherit;
```

Comment:
Super nitpicky: `overflow: inherit` strikes me as odd, is that on purpose? I would have expected `visible` explicitly, or maybe `unset` to say "go back to the default".

All of these do the same thing at the moment (computed result is `overflow: visible`) but `inherit` only works at the moment because the parent element, `<main>`, has `overflow: visible` (by default). I cannot think of many situations where we would really want to inherit a parent's overflow behaviour, it sounds like that would risk nested scrollbars and the like.

Reply:
Thanks for being nit-picky, there was no reason for setting `overflow: inherit;`.
Changed it. :)